spec/models/miq_product_feature_spec.rb

```diff
@@ -1,15 +1,16 @@
 require "spec_helper"
 
 describe MiqProductFeature do
-  before do
-    @expected_feature_count = 846
+  before(:all) do
+    MiqRegion.seed
+    MiqProductFeature.seed
+    @expected_feature_count = MiqProductFeature.count
+    MiqProductFeature.destroy_all
   end
 
   context ".seed" do
-    it "empty table" do
-      MiqRegion.seed
-      MiqProductFeature.seed
-      MiqProductFeature.count.should eq(@expected_feature_count)
+    it "sanity" do
+      @expected_feature_count.should_not eq(0)
```

Comment:
Line 20, becomes, get the count of rows after initial seed, seed again, verify the count matches after the first seed.